x-pack/plugins/security_solution/server/endpoint/services/actions/clients/endpoint/endpoint_actions_client.test.ts

```diff
@@ -261,6 +261,8 @@ describe('EndpointActionsClient', () => {
     execute: responseActionsClientMock.createExecuteOptions(getCommonResponseActionOptions()),
 
     upload: responseActionsClientMock.createUploadOptions(getCommonResponseActionOptions()),
+
+    scan: responseActionsClientMock.createUploadOptions(getCommonResponseActionOptions()),
```

Comment:
using incorrect mock here. Should create one that is specific for `scan` (even if the same options as upload are returned). Will prevent issues if we ever enhance one command to add more parameters, but not for the others that maybe reusing the same props

Reply:
Sheesh! Yeah I'll update this as well. Thanks for catching this.

Reply:
FYI: I think that for `isolate`/`release` I created two separate functions out of `responseActionsClientMock`, but internally In that mock file) its actually using the same payload for both. This ensure that consumers of the mock still use the appropriate mock creation function and if we need to ever change them to differ from each other, then only the mock provider internal logic needs to be changed.

Reply:
done 881a8bbd69c8f5792d258171949aed099b7e6584